examples/metrics/src/main/java/io/opentelemetry/example/metrics/LongMeterExample.java

```diff
@@ -0,0 +1,38 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.LongCounter.BoundLongCounter;
+import io.opentelemetry.metrics.Meter;
+import java.util.Random;
+
+/**
+ * Long meter example with bound counter
+ */
+public class LongMeterExample {
```

Comment:
This should be renamed to `LongCounterExample`, and it still needs to have a span context surrounding the usage of the counter, to be idiomatically correct.

The javadoc should definitely be more descriptive, and probably include a reference to the relevant section of the metric specifications. It should explain under what circumstances it is appropriate to use a LongCounter, and explain what "bound" means. My expectation is that a newbie who is looking at this code will see an idiomatic example of how to use the instrument being demonstrated and understand why and when they would use such an instrument.

Reply:
@jkwatson gotch you